source/includes/php/errors.md

```diff
@@ -0,0 +1,58 @@
+# Errors
+
+> Catch an Error from the library
+
+```php
+<?php
+require 'path/to/lib/shoprunback-php/init.php';
+
+\Shoprunback\RestClient::getClient()->setToken('yourApiToken');
+
+try {
+  $product = \Shoprunback\Elements\Product::retrieve('a wrong ID');
+} catch (\Shoprunback\Error\Error $e) {
+  // Here you can manage the error
+}
+```
+
+To prevent errors to crash your code, you must catch the exceptions.
+
+All the errors the library can return inherit from the class **\Shoprunback\Error\Error**. ->
+
+#### Parameters
+
+Parameter | Type | Description | Specific to
+-|-|-|-
+**message** | **String** | The mandatory message for an Exception | All
+**response** | **RestResponse** | The response from the server explaining why the call failed | **RestClientError**
+
+## Error types
+
+> Use a RestClientError
+
+```php
+<?php
+require 'path/to/lib/shoprunback-php/init.php';
+
+\Shoprunback\RestClient::getClient()->setToken('a wrong token');
```

Comment:
how do you handle exceptions on this call?

I'm guessing a wrong token returns an error as well?